src/layers/sds/domain/nhs_mhs.py

```diff
@@ -47,19 +47,20 @@ class NhsMhs(SdsBaseModel):
     nhs_mhs_duplicate_elimination: Optional[YesNoAnswer] = Field(
         alias="nhsmhsduplicateelimination"
     )
+    binding: Optional[str] = Field(alias="binding")
```

Comment:
Potentially don't ask for binding and always combine https:// with fqdn when generating the endpoint/address